### PR TITLE
fixed race condition

### PR DESCRIPTION
**Pull request by ikeikeikeike**
hi

I wanna make sure that passes race condition on go testing every time so that this PR is able to do that, and also fixed that race condition happened.

Thanks for your library.

### CODE REVIEW

File: mysqltest.go [hunk 1]
```diff
@@ -341,6 +341,8 @@ func (m *TestMysqld) Start() error {
 	go io.Copy(file, stderrpipe)
 
 	c := make(chan bool)
+
+	cmd.Start()
```
Comment (lestrrat) [inline mysqltest.go]: I don't get it. How does this help? `cmd.Run()` is being run in the goroutine. it looks as if you are executing `cmd` twice

Comment (lestrrat) [inline mysqltest.go]: Maybe you meant to replace `cmd.Run()` below with `cmd.Wait()` ?

Comment (ikeikeikeike) [inline mysqltest.go]: Ah it's actually correct. I wrote mistake that, and thanks for your noticed. However as you may know that there're still race condition in both methods between `TestMysqld.Start` and `TestMysqld.Stop` if it has replaced `cmd.Run` to `cmd.Wait` so I'd try to fix that.

**Comment by ikeikeikeike** [comment]
I think that this commit is the same meaning as previous code. How's this ?

**Comment by lestrrat** [comment]
Yes, it becomes equivalent, but now the problem is that my original code is just not go-ish at all :/

I'm  fixing this in a separate PR, as it will entail a much larger changeset.

**Comment by lestrrat** [comment]
Fixed by #7

**Pull request closed without merging**